.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
Could you modify the steps to fit the steps done in a terminal window? Maybe something like "open a terminal window", "type npm install" ...